src/api2/request/timeline.cpp

```diff
@@ -39,7 +38,7 @@ void Instagramv2::getUserFeed(QString userID, QString max_id, QString minTimesta
                      "&ranked_content=true";
 
 
-    InstagramRequestv2 *getUserFeedRequest = new InstagramRequestv2();
-    getUserFeedRequest->request(target,NULL);
+    InstagramRequestv2 *getUserFeedRequest =
```

Comment:
Any reason for a new line after the '=' char?

Reply:
These changes were applied automatically through a regular expression, and I reformatted the lines where I noticed that the formatting could be improved. I must have missed this one, I'll fix it.